presets/4.3/vtx/DJI_FPV-WTF_MSP-OSD.txt

```diff
@@ -0,0 +1,53 @@
+#$ TITLE: FPV.WTF MSP-OSD
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS:  vtx, dji, vista, air unit, fpv.wtf, wtfos, msp-osd, displayport, msp
+#$ AUTHOR: benlumley
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION:
+#$ DESCRIPTION: <img src="https://fpv.wtf/logo192.png" width="100" height="100" style="object-fit: cover; margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: # DJI FPV.WTF MSP-OSD setup
+#$ DESCRIPTION: Sets up MSP displayport on selected UART for use with MSP-OSD
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: The information provided on this preset is for educational and entertainment purposes only. Betaflight makes no representations as to the safety or legality of the use of any information provided herein. End users assume all responsibility and liability for ensuring they are complying with all relevant laws and regulations.
+
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/298
+
+set vcd_video_system = PAL
```

Comment:
Should this be an option as some like to use NTSC?

Reply:
It's not related to NTSC or PAL; it's just about making sure the configurator can see all 16 lines to configure.

Reply:
I think for hacked DJI it should always be PAL?

Reply:
correct. 
as i say - the only reason for this is to ensure the configurator shows 16 lines (NTSC only shows 15)

Reply:
Thanks!